### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by noxidsoft** [Phlow/responsive-one#1]
Hi from Australia,

Moritz, I would love to see an implementation of the Foundation off-canvas feature included in the template. I attempted with Feeling-Responsive fork of my own, but it doesn't work very well. It would be great as an optional and/or feature in line with the current main menu. I am going to continue working on my own version/fork anyway.

What do you think?

Kind regards,
Noel

**Comment by noxidsoft** [comment]
Similar to this:
[http://calebserna.com/demo/off-canvas-topbar/](http://calebserna.com/demo/off-canvas-topbar/)

or this (desired mobile first)...
[http://codepen.io/winghouchan/pen/Aqbpz/](http://codepen.io/winghouchan/pen/Aqbpz/)

**Comment by Phlow** [comment]
Hi Noel!

I cut down the javascript and I didn't include the off-canvas-feature. That's why it's not working. I don't want to bloat *Feeling Responsive* to much. I care about page weight to make it as fast and leight as possible.

What I can do for you: I can send you a version of the javascript needed to make the off-canvas menue going to work. The rest is covered by the [foundation documentation](http://foundation.zurb.com/docs/components/offcanvas.html).

Interested?

Greets, Moritz

**Comment by noxidsoft** [comment]
Yes, most certainly, that is very generous of you. Do you have a version that includes the whole of Foundation in the implementation, a few hundred thousandths of a second increase in page load is okay for a project I want to convert over from Bootstrap 2? It will be much faster than the current instance anyway.

I am wanting to use a version of your Jekyll responsive-one or similar over here (world-wide community project): [HazAlerts.com.au](http://www.hazalerts.com.au)

**Comment by Phlow** [comment]
Hi Noel!

I added it to my theme:

https://github.com/Phlow/responsive-one/tree/gh-pages/assets/js

There you'll find 

https://github.com/Phlow/responsive-one/blob/gh-pages/assets/js/javascript.fullversion.js

and the minified version.

To use it, you have to change `https://github.com/Phlow/responsive-one/blob/gh-pages/_includes/footer_scripts`

    <script src="{{ site.url }}/assets/js/javascript.fullversion.min.js"></script>

Then it should work.
